arduino_mkrzero/.cargo/config

```diff
@@ -0,0 +1,10 @@
+# samd21 is a Cortex-M0 and thus thumbv6m
```

Comment:
can we survive with the top level `.cargo/config`, or alternatively, should each of the board crates have one of these?

Reply:
I pushed 39874f8b5b9f1e9f6862f1910034a62e112f334f with the net effect of this at the top level, so I think you can remove this file from your PR.  (Also, I only just noticed that you changed the top level file in this PR too)

Thanks!